observations/atmosphere/aircraft.yaml.j2

```diff
@@ -7,146 +7,538 @@
     obsdatain:
       engine:
         type: H5File
-        obsfile: "{{atmosphere_obsdatain_path}}/{{atmosphere_obsdatain_prefix}}{{observation_from_jcb}}{{atmosphere_obsdatain_suffix}}"
+        obsfile: "/work2/noaa/da/esposito/ufoeval/GSIobserver_NE_ADC/2021080100/ioda/obs/aircraft_obs_2021080100.nc4" #NICKE test
```

Comment:
I think these should not be hard coded, right?

Reply:
Oops, missed that. Deleting now.